.github/workflows/windows.yml

```diff
@@ -491,6 +473,11 @@ jobs:
           curl --disable --fail --silent --show-error --connect-timeout 15 --max-time 60 --retry 3 https://live.sysinternals.com/handle64.exe --output /bin/handle64.exe
           python3 -m pip --disable-pip-version-check --no-input --no-cache-dir install --progress-bar off --prefer-binary impacket
 
+      - name: 'downgrade msys2-runtime'
+        if: ${{ matrix.tflags != 'skipall' && matrix.tflags != 'skiprun' }}
+        timeout-minutes: 2
+        run: exec /usr/bin/pacman --noconfirm --noprogressbar --upgrade https://mirror.msys2.org/msys/x86_64/msys2-runtime-3.5.4-8-x86_64.pkg.tar.zst
```

Comment:
```suggestion
        run: |
          /usr/bin/sed -i -e 's/^CheckSpace/#CheckSpace/' /etc/pacman.conf
          exec /usr/bin/pacman --noconfirm --noprogressbar --upgrade https://mirror.msys2.org/msys/x86_64/msys2-runtime-3.5.4-8-x86_64.pkg.tar.zst
```

?

Reply:
yeah, I see [9b1a2d0](https://github.com/curl/curl/pull/16424/commits/9b1a2d0ae0d8cc711393987156b6e81b0093cf32) but msys2's /usr/bin wasn't on the path there, you were probably getting the sed from the default gfw install, who doesn't have /etc/pacman.conf in its root.  Confusing, huh?

Reply:
Let's try that quick! (The downgrade is already quicker than the GfW install.)

Thanks for your tips, it'd be nice to install into the default path. I'll try it.

Reply:
`sed` tests OK with the abs path.